packages/sysweb3-keyring/src/keyring-manager.ts

```diff
@@ -243,77 +244,49 @@ export const KeyringManager = (): IKeyringManager => {
     };
   };
 
-  const _setDerivedWeb3Accounts = async (id: number, label: string) => {
+  const _setDerivedWeb3Accounts = async (id: number) => {
```

Comment:
The label was added here because this function is also used by  `_getLatestUpdateForWeb3Accounts` which is called by  `getLatestUpdateForAccount` and this causes the label if any chosen by the user to be overwritten. So removing the label here will bring back a bug.